### ISSUE TITLE
Relative JSON Pointers

### ISSUE THREAD

**Issue by Julian** [stefankoegl/python-json-pointer#34]
Hi!

Was wondering what you think about @handrews' [Relative JSON Pointer](https://tools.ietf.org/html/draft-handrews-relative-json-pointer-01) specification.

Do you feel it has a place within this library, or would you rather it be separate?

(I ask of course because as in #20, I'm about to add support for validating relative JSON pointers to `jsonschema`).

Cheers,
-J

**Comment by steinymity** [comment]
As far as I can see from the specs, relative JSON pointers and ordinary JSON pointers can clearly be distinguished based on their first character, so an extension to this library should be possible.

**Comment by handrews** [comment]
@steinymity yes the intention of the original author (I just took it over to update it- it will get a new draft shortly which fixes a typo or two but nothing major) was that they were never ambiguous.
